### DISCUSSION TITLE
Msal clientSecret config

### DISCUSSION THREAD

**Discussion by rscflexible** [milestonetg/msal-js-dart#42] [Q&A] [answered]
Could you please provide me with an example of how to configure the clientSecret for a PublicClientApplication?

**Comment by Francessco121** [comment, marked as answer]
Public client applications do not support client secrets. In MSAL terms, you may be thinking of confidential client applications, however msal.js is a browser-only library and as such does not support server-side confidential client applications. MSDN explains the difference here: https://learn.microsoft.com/en-us/azure/active-directory/develop/msal-client-applications.
